### PR TITLE
[Attention OCR] Document model support

### PR DESCRIPTION
**Pull request by evolosen**
Add instruction to convert [Attention OCR](https://github.com/emedvedev/attention-ocr) model from TensorFlow* to IR format.

Ticket #39480

### CODE REVIEW

**Review by vgavrilo — APPROVED** [review]

**Comment by achetver** [comment]
Looks good. Please, add link to your instruction to `docs/MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md` in end of **Other supported topologies** section (lines 157-163). Sorry, I forgot to tell it before.

**Review by achetver — APPROVED** [review]

**Review by iimironov — APPROVED** [review]

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Convert TensorFlow* Attention OCR Model to Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_AttentionOCR_From_Tensorflow}
+
+This tutorial explains how to convert the Attention OCR (AOCR) model from the [TensorFlow* Attention OCR repository](https://github.com/emedvedev/attention-ocr) to the Intermediate Representation (IR).
+
+## Extract model from `aocr` library
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md:5]:
```suggestion
## Extract Model from `aocr` Library
```

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+The easiest way to get AOCR model is to download python library `aocr`:
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md:7]:
```suggestion
The easiest way to get an AOCR model is to download  `aocr` Python\* library:
```

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+```
+pip install git+https://github.com/emedvedev/attention-ocr.git@master#egg=aocr
+```
+This library contains pretrain model and allows to train and run AOCR using command line. After installing `aocr` you can extract model:
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md:11]:
```suggestion
This library contains a pretrained model and allows to train and run AOCR using the command line. After installing `aocr`, you can extract the model:
```

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+```
+aocr export --format=frozengraph model/path/
+```
+After this step you can find model in model/path/ folder.
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md:15]:
```suggestion
After this step, you can find the model in model/path/ folder.
```

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+## Convert the TensorFlow* AOCR Model to IR
+
+The original AOCR contains data preprocessing which consists of folowing steps:
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md:19]:
```suggestion
The original AOCR contains data preprocessing which consists of the following steps:
```

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+* Decoding input data to binary format where input data is image as string.
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md:20]: where input data is image as string 
maybe 
where input data is an image represented as a string

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+* Resizing binary image to working resolution.
+
+After that resized image is sent to CNN. The Model Optimizer does not support decoding images so you should exclude preprocessing from model converting.
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md:23]:
```suggestion
After that, the resized image is sent to the convolution neural network (CNN). The Model Optimizer does not support image decoding, so you should exclude preprocessing from model converting. 
```

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md (hunk 6 above)
Comment (evolosen) [reply docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md:20]: yes, as you said.

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md (hunk 2 above)
Comment (evolosen) [reply docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md]: Done

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md (hunk 7 above)
Comment (evolosen) [reply docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md]: Done

**Review by tsavina — APPROVED** [review]

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
+# Convert TensorFlow* Attention OCR Model to Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_AttentionOCR_From_Tensorflow}
+
+This tutorial explains how to convert the Attention OCR (AOCR) model from the [TensorFlow* Attention OCR repository](https://github.com/emedvedev/attention-ocr) to the Intermediate Representation (IR).
```
Comment (lazarevevgeny) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md:3]: please, specify commit hash which you tested your code against

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+python3 path/to/model_optimizer/mo_tf.py \
+--input_model=model/path/frozen_graph.pb \
+--input="map/TensorArrayStack/TensorArrayGatherV3:0[1 32 86 1]" \
+--output "transpose_1,transpose_2"
+--output_dir path/to/ir/
+```
+
+Where:
+* `map/TensorArrayStack/TensorArrayGatherV3:0[1 32 86 1]` - result of data preprocessing that was cut.
+* `transpose_1,transpose_2` - predictions (int array) and their probabilities (float array).
```
Comment (lazarevevgeny) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md:34]: Change to something like this:
```
* `transpose_1` - name of the node producing tensor with predicted characters.
* `transpose_2` - name of the node producing tensor with predicted characters probabilities.
```

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+```
+After this step you can find the model in model/path/ folder.
+
+## Convert the TensorFlow* AOCR Model to IR
+
+The original AOCR contains data preprocessing which consists of the following steps:
+* Decoding input data to binary format where input data is an image represented as a string.
+* Resizing binary image to working resolution.
+
+After that, the resized image is sent to the convolution neural network (CNN). The Model Optimizer does not support image decoding so you should exclude preprocessing from model converting.
```
Comment (lazarevevgeny) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md:23]: May be change the final sentence to: `The Model Optimizer does not support image decoding so you should cut-of preprocessing part of the model using "--input" command line parameter`.

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+The original AOCR contains data preprocessing which consists of the following steps:
+* Decoding input data to binary format where input data is an image represented as a string.
+* Resizing binary image to working resolution.
+
+After that, the resized image is sent to the convolution neural network (CNN). The Model Optimizer does not support image decoding so you should exclude preprocessing from model converting. 
+```sh
+python3 path/to/model_optimizer/mo_tf.py \
+--input_model=model/path/frozen_graph.pb \
+--input="map/TensorArrayStack/TensorArrayGatherV3:0[1 32 86 1]" \
+--output "transpose_1,transpose_2"
```
Comment (lazarevevgeny) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md:28]: Missing "\\" in the end of the string

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+```
+This library contains a pretrained model and allows to train and run AOCR using the command line. After installing `aocr`, you can extract the model:
+```
+aocr export --format=frozengraph model/path/
+```
+After this step you can find the model in model/path/ folder.
+
+## Convert the TensorFlow* AOCR Model to IR
+
+The original AOCR contains data preprocessing which consists of the following steps:
```
Comment (lazarevevgeny) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md:19]: `original AOCR contains` -> `original AOCR model contains`

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**